native/packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBar-test.android.js

```diff
@@ -0,0 +1,137 @@
+import renderer from 'react-test-renderer';
+import { View } from 'react-native';
+
+import BpkNavigationBar from './BpkNavigationBar.android';
+import BpkNavigationBarButtonAndroid from './BpkNavigationBarButtonAndroid.android';
+
+jest.mock('react-native', () => {
+  const reactNative = jest.requireActual('react-native');
+  jest
+    .spyOn(reactNative.Platform, 'select')
+    .mockImplementation(obj => obj.android || obj.default);
+  reactNative.Platform.OS = 'android';
+  reactNative.TouchableNativeFeedback.SelectableBackground = jest.fn();
+
+  return reactNative;
+});
+
+jest.mock('TouchableNativeFeedback', () =>
+  jest.requireActual(
+    'react-native/Libraries/Components/Touchable/TouchableNativeFeedback.android.js',
+  ),
+);
+
+jest.mock('./BpkNavigationBarButtonAndroid.android', () =>
+  jest.requireActual('./BpkNavigationBarButtonAndroid.android.js'),
+);
+
+jest.mock(
+  './../node_modules/react-native-bpk-component-text/node_modules/bpk-tokens/tokens/base.react.native',
+  () => jest.requireActual('bpk-tokens/tokens/base.react.native.android.js'),
+);
+
+jest.mock('bpk-tokens/tokens/base.react.native', () =>
+  jest.requireActual('bpk-tokens/tokens/base.react.native.android.js'),
+);
+
+describe('android', () => {
+  describe('BpkNavigationBar', () => {
+    it('should render correctly', () => {
+      const tree = renderer
+        .create(
+          <BpkNavigationBar
+            title="Backpack"
+            leadingButton={
+              <BpkNavigationBarButtonAndroid
+                title="Back"
+                icon="long-arrow-left"
+                onPress={jest.fn()}
+              />
+            }
+          />,
+        )
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly with trailing button', () => {
+      const tree = renderer
+        .create(
+          <BpkNavigationBar
+            title="Backpack"
+            leadingButton={
+              <BpkNavigationBarButtonAndroid
+                title="Close"
+                icon="close"
+                onPress={jest.fn()}
+              />
+            }
+            trailingButton={
+              <BpkNavigationBarButtonAndroid
+                title="Done"
+                icon="tick"
+                onPress={jest.fn()}
+              />
+            }
+          />,
+        )
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly with a title view', () => {
+      const tree = renderer
+        .create(
+          <BpkNavigationBar
+            title={<View testID="title-view" />}
+            leadingButton={
+              <BpkNavigationBarButtonAndroid
+                title="Back"
+                icon="long-arrow-left"
+                onPress={jest.fn()}
+              />
+            }
+          />,
+        )
+        .toJSON();
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly with a subtitle view', () => {
```

Comment:
This test is no different to the one one line 101

Reply:
This has been fixed since I believe